UPGRADING

```diff
@@ -130,6 +130,36 @@ PHP 8.4 UPGRADE NOTES
 4. Deprecated Functionality
 ========================================
 
+- Date:
+  . Calling DatePeriod::__construct(string $isostr, int $options = 0). Use
+  DatePeriod::createFromISO8601String() instead.
+
+- Intl:
+  . Calling IntlCalendar::set() with more than 2 arguments and intlcal_set().
+    Use either IntlCalendar::setDate() or IntlCalendar::setDateTime() instead.
+  . Calling IntlGregorianCalendar::__construct() with more than 2 arguments
+    and intlgregcal_create_instance(). Use
+    IntlGregorianCalendar::createFromDate() or
+    IntlGregorianCalendar::createFromDateTime() instead.
+
+- LDAP:
+  . Calling ldap_connect() with more than 2 arguments. Use
+    ldap_connect_wallet() instead.
+  . Calling ldap_exop() with more than 4 arguments is deprecated. Use
+    ldap_exop_sync() instead.
+
+- PgSQL:
+  . Calling pgsql_fetch_result() with 2 arguments. Use the 3-parameter
```

Comment:
nit: I would suggest to clarify statements e.g. `Calling pgsql_fetch_result() with 2 arguments is deprecated` across the board as you did above with LDAP.